src/ui/search/searchKindBuilder.re

```diff
@@ -10,7 +11,7 @@ let options = [
 [@react.component]
 let make = (~value, ~on_change) => {
   <div className="mb-3">
-    <Label> {"Type" |> React.string} </Label>
+    <Label> ...{"Type" |> React.string} </Label>
```

Comment:
Is there no more concise way to do this? Seems overkill for things that are constants?

Reply:
The call to `React.string` is required to convert the string to a React element. I am however not entirely sure why the spreading operator is required here. It is not used in the jsoo-react [Hello World example](https://ml-in-barcelona.github.io/jsoo-react/hello-world-reason) and React.string should return just one element, but when one removes it, a type error is thrown (expected React.element but found 'a list).